runtime/convertValues_test.go

```diff
@@ -3770,72 +3763,6 @@ func TestRuntimePublicKeyImport(t *testing.T) {
 		cadence.NewUInt8(2),
 	})
 
-	t.Run("Test IsValid", func(t *testing.T) {
```

Comment:
Don't remove the test but keep it and just remove the read of `isValid` in the body. This tests also asserts if import of public keys properly validates them, not just that `isValid` returns the appropriate value